public/app/plugins/datasource/postgres/plugin.json

```diff
@@ -19,6 +19,7 @@
   "alerting": true,
   "annotations": true,
   "metrics": true,
+  "logs": true,
```

Comment:
@aocenas does this do anything meaningful?  If so I assume it should be true for any datasource that can return strings

Reply:
I think this is not really used nowadays. The only place I found is the datasource picker but maybe @aocenas knows better.

Reply:
I've talked with @aocenas and he confirmed that we only use this in the datasource picker.